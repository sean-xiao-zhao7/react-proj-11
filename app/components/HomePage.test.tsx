test("Renders with text Home Page");
