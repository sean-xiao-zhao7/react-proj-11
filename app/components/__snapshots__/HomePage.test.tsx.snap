// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page renders HomePage unchanged 1`] = `
<div>
  <div>
    <h1>
      Home Page
    </h1>
    <p>
      Test val: 
      False
    </p>
    <button>
      Change test
    </button>
  </div>
</div>
`;
